anagram/tests/anagram.rs

```diff
@@ -1,11 +1,6 @@
-#![crate_name = "anagram_test"]
-#![crate_type = "lib"]
-#![feature(collections)]
-
-mod anagram;
+extern crate anagram;
 
 #[test]
-#[ignore]
```

Comment:
The languages are not consistent at all here, and I don't necessarily think that we should strive for consistency in this respect.

The javascript and ruby tracks have a lot more beginners than other tracks, and beginners tend to get overwhelmed by seeing a lot of failures.

The Go track is my favorite in terms of the behavior of the test suite. It generally uses table-tests and fails fast so that you get only the first failure, but you don't have to manually change the test suite to get the next test.